### PR TITLE
Cleanup remote debugger code

### PR DESCRIPTION
**Pull request by SaveTheRbtz**

### CODE REVIEW

File: utils/debug/registerCache.go [hunk 1]
```diff
@@ -135,17 +135,5 @@ func (c *fileRegisterCache) Persist() error {
 	if err != nil {
 		return err
 	}
-	err = f.Sync()
-	if err != nil {
-		return err
```
Comment (ramtinms) [inline utils/debug/registerCache.go:140]: should we keep one of the syncs here ?

**Review by ramtinms — APPROVED** [review]
I think these were introduced by some sort of merge issue.

File: utils/debug/registerCache.go (hunk 1 above)
Comment (SaveTheRbtz) [reply utils/debug/registerCache.go:140]: bufio.Writer does require calling `Flush`, everything else does that automagically on Close.

**Review by pattyshack — APPROVED** [review]

**Comment by SaveTheRbtz** [comment]
bors merge

**Pull request merged**